### DISCUSSION TITLE
Multilingual document support

### DISCUSSION THREAD

**Discussion by n2ptune** [alpinejs/alpine#1789] [2. Show and tell]
Hello,

I recently saw a nice updated document. So i want to share this with the people of my country. However, there are people who are not used to English.

Is there plan to support multiple languages? If there is such a plan, I would appreciate it if you could suggest a direction for me to contribute in project.

**Comment by n2ptune** [comment]
Oh, i found `packages/docs` directory after written discussion....

then, can i pull request for translated document this directory? is there any convention?

**Reply by Pronian** [reply]
It looks like there already is a PR for Indonesian: https://github.com/alpinejs/alpine/pull/1533

So it looks like it shouldn't be a problem to submit a new one for a new language
